migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -383,7 +382,7 @@ initial migration, and the longer the buffered files need to be stored.
 <LiveMigrationRoles />
 
 ### 4.b Dump the database schema from the source database
-<DumpSourceSchema />
+<DumpPreDataSourceSchema />
 
 ### 4.c Load the roles and schema into the target database
 ```sh
```

Comment:
The block that starts here looks like it's missing the change from `schema.sql` to `pre-data-dump.sql`.